data/net/minecraft/server/rcon/thread/RconClient.mapping

```diff
@@ -8,3 +12,4 @@ CLASS net/minecraft/server/rcon/thread/RconClient
 		COMMENT Sends the standard RCon 'authorization failed' response packet
 	METHOD sendCmdResponse (ILjava/lang/String;)V
 		COMMENT Splits the response message into individual packets and sends each one
+		ARG 1 exception
```

Comment:
Looking at the [docs for the Source RCON Protocol](https://developer.valvesoftware.com/wiki/Source_RCON_Protocol), which is the protocol that Minecraft implements for RCON, this value is transmitted as the Packet ID. See the implementation of the method, which calls `send(int, int, String)`, where the parameters are the Packet ID, Packet Type (in this case, `0` for `SERVERDATA_RESPONSE_VALUE`) and the Body (the response of the server).

```suggestion
		ARG 1 id
```